partner_communication/static/src/less/communication_dashboard.less

```diff
@@ -0,0 +1,223 @@
+            // give space for graph section at the bottom
+            margin-bottom: 75px;
+            min-height: 120px;
+        }
+
+        .o_kanban_margin_top {
+            margin-top: 20px;
+        }
+
+        .o_kanban_graph_section {
+            .o-position-absolute(auto, 0, 0, 0);
+            .o_graph_linechart {
+                > svg {
+                    height: 20px;
+
+                    // X axis
+                    g.nv-x.nv-axis {
+                        transform: translateY(20px);
+
+                        // Remove max and min labels
+                        g.nv-axisMaxMin {
+                            display: none;
+                        }
+                        g.tick.major {
+                            // Don't know why, but leads to strange bold text
+                            // stroke: @odoo-brand-primary;
+
+                            line {
+                                display: none;
+                            }
+                        }
+                    }
+
+                    // !important are needed because nvd3 uses inline style on elements
+
+                    // Remove strokes
+                    g.nv-linesWrap {
+                        g.nv-group.nv-series-0 {
+                            fill-opacity: 0.3 !important;
+                            fill: @odoo-brand-primary !important;
+                            stroke: @odoo-brand-primary !important;
+
+                            .nv-point {
+                                visibility: hidden;
+                                r: 3;
+                                stroke: @odoo-brand-primary !important;
+                                fill: white;
+                                stroke-width: 2;
+                            }
+                            .nv-point:nth-child(5n+1) {
+                                visibility: visible;
+                                fill-opacity: .95 !important;
+                                stroke-opacity: .95 !important;
+                            }
+                        }
+                    }
+                }
+            }
+            .o_graph_barchart {
+                > svg {
+                    height: 20px;
+                    overflow: visible;
+                    g.nv-barsWrap {
+                        g.nv-group.nv-series-0 {
+                            g.past {
+                                opacity: 0.5;
+                                fill: @odoo-brand-primary !important;
+                            }
+                            g.future {
+                                opacity: 0.5;
+                                fill: @odoo-brand-optional !important;
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        .o_invisible .o_kanban_graph_section {
+            display: none; // This fix a bug on firefox
+        }
+    }
+
+    .o_communication_dashboard {
```

Comment:
Can't you use another existing class and add your custom class which will just override what you need to change?
Then in your template your dashboard will have two classes : `o_existing_dashboard` (replace existing by some real class), and `my_custom_communication_class`.